blueocean-dashboard/src/main/js/OrganizationPipelines.jsx

```diff
@@ -132,10 +121,17 @@ class OrganizationPipelines extends Component {
      components and get rid of the seperate connect in each subcomponents -> see RunDetailsPipeline
      */
     render() {
-        if (!this.props.allPipelines && !this.props.organizationPipelines) {
+        const { allPipelines, organizationPipelines } = this.props;
+        let pipelines = null;
+        if (!allPipelines && !organizationPipelines) {
             return null;
         }
-        return this.props.children;
+        if (allPipelines && allPipelines.$success) {
+            pipelines = allPipelines;
+        } else if (organizationPipelines && organizationPipelines.$success) {
+            pipelines = organizationPipelines;
+        }
```

Comment:
So reading the above code .... both `allPipelines` and `organizationPipelines` cannot both be non-null at the same time, or does `allPipelines` always take precedence when they're both non-null?